specs.js

```diff
@@ -0,0 +1,153 @@
+/* global Given, Then, describe */
+var expect = require('chai').expect
+var nock = require('nock')
+
+describe('when using trimperson api', () => {
+  var trim
+  var mockURL
+  var mockToken
+
+  Given(() => mockURL = 'http://localhost')
+  Given(() => mockToken = 'DEADBEEF')
+
+  Given(() => {
```

Comment:
this looks like the "When" to me. The action being creating/setting up the TRIM object